### ISSUE TITLE
Feature request: Single Chore Multiple Users

### ISSUE THREAD

**Issue by FracKenA** [grocy/grocy#1415]
As a user of grocy, I would like the ability to assign a chore to multiple users at EVERY interval. 

Example.

Chore Name - Make your bed. Frequency Daily.
     Assigned to Child 1
     Assigned to Child 2
     Assigned to Child 3

It appears on each Childs list daily.

**Comment by Ape** [comment]
You can just create three individual tasks.

**Comment by FracKenA** [comment]
Yes, that is what I am doing now, however it is a very dirty and hard to manage with 5 kids.

**Comment by Ubersjaak** [comment]
I would like this as well/ kinda similar. Usecase: We have cats, cats need food. Doesn't matter who feeds them, as long as it is done

**Comment by berrnd** [comment]
> Basicly add a assignment type: "multiple people" for instance

One topic per request (as mentioned on the feature request template), everything else is untrackable. So moved to #1935.

**Comment by foxhoundv** [comment]
I don't know if this was covered in one of the multiple side issues related to this. Since this seems like the main issue I will comment it here. I really like the feature for chores that gives the option to assign the task to who does it the least. As I am sure most parents go through the same, "I did it last" issue from all their children if not most of them, lol. Since my wife and I have our own account and these chores that are specific to our children, I can see grouping them as a great idea, not to mention if the option to assign it to who does it the least can be applied to the group itself. Maybe even an exclude section to be able to choose to exclude certain users? Just throwing out ideas!
